### PR TITLE
Add file argument to get_nml_value

### PR DESCRIPTION
**Pull request by jsta**
See #233

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -42,6 +42,7 @@ export(write_nml)
 import(GLMr)
```
Comment (jread-usgs) [inline NAMESPACE]: I think keeping this `import(GLMr)` swamps the function-specific import you added below. Or at least that is my understanding of the namespacing

Comment (jsta) [inline NAMESPACE]: Right. I made sure to make all imports specific in 9375e77

**Comment by jsta** [comment]
@jread-usgs Incremented and good to go.

**Comment by jread-usgs** [comment]
Nice, sorry - didn't see this until now

**Pull request merged**